review/communication-react.api.md

```diff
@@ -0,0 +1,1828 @@
+    // (undocumented)
+    onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
+    onRenderAvatar?: OnRenderAvatarCallback;
+    onRenderLocalVideoTile?: (localParticipant: VideoGalleryLocalParticipant) => JSX.Element;
+    onRenderRemoteVideoTile?: (remoteParticipant: VideoGalleryRemoteParticipant) => JSX.Element;
+    remoteParticipants?: VideoGalleryRemoteParticipant[];
+    remoteVideoViewOption?: VideoStreamOptions;
+    showMuteIndicator?: boolean;
+    styles?: BaseCustomStylesProps;
+}
+
+// @public
+export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
+    isSpeaking?: boolean;
+    screenShareStream?: VideoGalleryStream;
+}
+
+// @public (undocumented)
+export const videoGallerySelector: OutputParametricSelector<CallClientState, CallingBaseSelectorProps, {
+screenShareParticipant: VideoGalleryRemoteParticipant | undefined;
+localParticipant: {
+userId: string;
+displayName: string;
+isMuted: boolean | undefined;
+isScreenSharingOn: boolean | undefined;
+videoStream: {
+isAvailable: boolean;
+isMirrored: boolean | undefined;
+renderElement: HTMLElement | undefined;
+};
+};
+remoteParticipants: VideoGalleryRemoteParticipant[];
+}, (res1: string | undefined, res2: {
+[keys: string]: RemoteParticipantState;
+} | undefined, res3: LocalVideoStreamState[] | undefined, res4: boolean | undefined, res5: boolean | undefined, res6: string | undefined, res7: string) => {
+screenShareParticipant: VideoGalleryRemoteParticipant | undefined;
+localParticipant: {
+userId: string;
+displayName: string;
+isMuted: boolean | undefined;
+isScreenSharingOn: boolean | undefined;
+videoStream: {
+isAvailable: boolean;
+isMirrored: boolean | undefined;
+renderElement: HTMLElement | undefined;
+};
+};
+remoteParticipants: VideoGalleryRemoteParticipant[];
+}>;
+
+// @public
+export interface VideoGalleryStream {
+    id?: number;
+    isAvailable?: boolean;
+    isMirrored?: boolean;
+    renderElement?: HTMLElement;
+}
+
+// @public
+export interface VideoStreamOptions {
+    isMirrored?: boolean;
+    scalingMode?: 'Stretch' | 'Crop' | 'Fit';
+}
+
+// @public
+export interface VideoStreamRendererViewState {
+    isMirrored: boolean;
+    scalingMode: ScalingMode;
+    target: HTMLElement;
+}
+
+// @public (undocumented)
+export const VideoTile: (props: VideoTileProps) => JSX.Element;
+
+// @public
+export interface VideoTileProps {
+    children?: React_2.ReactNode;
+    displayName?: string;
+    isMirrored?: boolean;
+    isMuted?: boolean;
+    noVideoAvailableAriaLabel?: string;
+    onRenderPlaceholder?: OnRenderAvatarCallback;
+    renderElement?: JSX.Element | null;
+    showMuteIndicator?: boolean;
+    styles?: VideoTileStylesProps;
+    userId?: string;
+}
+
+// @public (undocumented)
+export interface VideoTileStylesProps extends BaseCustomStylesProps {
+    displayNameContainer?: IStyle;
+    overlayContainer?: IStyle;
+    videoContainer?: IStyle;
+}
+
+// (No @packageDocumentation comment for this package)
```

Comment:
we should add package documentation for this package as its our primary export

Reply:
Since you are the author for this package, do you have idea of package documentation? I will leave this as an open question since it is not necessary for azure review though

Reply:
Not sure what is expected here but I'll assign this to myself